test_Levon.py

```diff
@@ -0,0 +1,2 @@
+if 1>0:
```

Comment:
Change your function. Create a function that will print your name, surname and favoritw car model.